src/examples/Logic_fastaSearch.py

```diff
@@ -0,0 +1,225 @@
+
+import webbrowser
+
+
+class logic:
+
+    # Zusammenstellung des dictionaries und der 3 Listen zum suchen für die a) und b)
+    def protein_dictionary(fastaFile):
+        """Gets fasta File Path and saves all infromations about the proteins 
+        in separate lists 
+
+        Parameters
+        ----------
+        fastaFile : path to fasta file
+
+        Returns
+        -------
+        for all proteins either Decoy or normal Lists/ dectionary.
+        """
+        dictKeyAccession = {}
+        proteinList = []
+        proteinNameList = []
+        proteinOSList = []
+        dictKeyAccessionDECOY = {}
+        proteinListDECOY = []
+        proteinNameListDECOY = []
+        proteinOSListDECOY = []
+        counter = 0
+        with open(fastaFile) as file_content:
+            for seqs in file_content:
```

Comment:
more comments required... e.g. for the for-loop some high-level description what happens inside the loop